extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/QuantityScrutinizerTest.java

```diff
@@ -24,10 +42,17 @@
             new BigDecimal("1.545"));
     
     private QuantityValue wrongUnitValue = Datamodel.makeQuantityValue(
-            new BigDecimal("1.234"), "Q346721");
+            new BigDecimal("2"), "Q346721");
```

Comment:
Why did you need to change this value? Isn't this changing the meaning of the tests below?